cmd/cluster_kubeconfig.go

```diff
@@ -45,41 +53,153 @@ Example 4: hetzner-kube cluster kubeconfig -n my-cluster -p > my-conf.yaml # pri
 
 		FatalOnError(err)
 
-		printContent, _ := cmd.Flags().GetBool("print")
-		force, _ := cmd.Flags().GetBool("force")
-
-		if printContent {
-			fmt.Println(kubeConfigContent)
+		// get sanitized kubeconfig
+		// we need isSanitized flag to ensure we do not want do a merge if we fail to sanitize config
+		// if it fails we simply print out config as it is
+		isSanitized := false
+		newKubeConfig, err := sanitizeKubeConfig(kubeConfigContent, provider.GetCluster().Name, "hetzner")
+		if err != nil {
+			log.Printf("KubeConfig sanitise process failed, default config will be used instead. Error: %s", err.Error())
 		} else {
-			fmt.Println("create file")
+			kubeConfigContent = newKubeConfig
+			isSanitized = true
+		}
 
-			usr, _ := user.Current()
-			dir := usr.HomeDir
-			path := fmt.Sprintf("%s/.kube", dir)
+		if merge, _ := cmd.Flags().GetBool("merge"); merge && isSanitized {
```

Comment:
Ignored error, I know that some command ignore error from flag, but I think is not a good practice, WDYT?

Reply:
If you are sure you properly defined the flag, you can easily ignore this error. I did this several times. The error which might occur here is if the flag itself doesn't exist